src/cocoa/toga_cocoa/libs/webkit.py

```diff
@@ -11,5 +11,5 @@
 ######################################################################
 
 ######################################################################
-# WebView.h
-WebView = ObjCClass('WebView')
+# WKWebView.h
+WKWebView = ObjCClass('WKWebView')
```

Comment:
At column 13: (F405) 'ObjCClass' may be undefined, or defined from star imports: ctypes, rubicon.objc